spec/helpers/handle_helper_spec.rb

```diff
@@ -1,21 +1,20 @@
 require "spec_helper"
 include HandleHelper
 
-describe HandleHelper, unless: $in_travis do
+describe HandleHelper do
   let(:bills_file) { FactoryGirl.create(:bills_complete_file) }
-  `mysql -u "#{ENV["HANDLE_TEST_USERNAME"]}" < "#{Rails.root}"/spec/fixtures/files/handlesTEST.sql`
```

Comment:
@elizoller These need to stay.

If you use non-test values in specs, and they were run on production, you'd be adding test values to the production handle database.

By putting in the values in Travis.yml, you're absolving yourself of that risk though.